watson-developer-cloud/discovery-components
# feat: PDF view, single tooltip, show when hovering/selecting an enrichment

#### What do these changes do/fix?

Tooltips for enrichments is implemented in components. It is accessed through the discovery tooling:

<img width="1540" alt="image" src="https://github.com/watson-developer-cloud/discovery-components/assets/45341518/2ab72403-77ad-4792-8288-1fc4ebe53638">


#### How do you test/verify these changes?

1. Create project with trained SDU model. This creates the text mappings for the PDF view. 
2. Search for documents, select a document
3. Open the advanced view
4. Select one or more enrichments.
5. Mouse over enrichments to see tooltips.

#### Have you documented your changes (if necessary)?

In the code

#### Are there any breaking changes included in this pull request?

No
